modules/op-configuring-pipelines-as-code-for-a-github-app.adoc

```diff
@@ -33,11 +33,11 @@ To create and configure a GitHub App manually for {pac}, perform the following s
 
 * **GitHub Application Name**: `OpenShift Pipelines`
 * **Homepage URL**: OpenShift Console URL 
-* **Webhook URL**: The {pac} route or ingress URL. You can find it by executing the command `echo https://$(oc get route -n pipelines-as-code el-pipelines-as-code-interceptor -o jsonpath='{.spec.host}')`.
+* **Webhook URL**: The {pac} route or ingress URL. You can find it by executing the command `echo https://$(oc get route -n openshift-pipelines pipelines-as-code-controller -o jsonpath='{.spec.host}')`.
 +
 [NOTE]
 ====
-For {pac} installated by default using the {pipelines-title} Operator, use the `openshift-pipelines` namespace instead of `pipelines-as-code`.
+For {pac} installated by default using the {pipelines-title} Operator, use the `pipelines-as-code` namespace instead of `openshift-pipelines`.
```

Comment:
If pac is installed directly without using the Operator then use `pipelines-as-code` instead of `openshift-pipelines`